letsencrypt/cli.py

```diff
@@ -560,13 +563,10 @@ def read_file(filename, mode="rb"):
     :returns: A tuple of filename and its contents
     :rtype: tuple
 
-    :raises argparse.ArgumentTypeError: File does not exist or is not readable.
+    :raises IOError: File does not exist or is not readable.
 
     """
-    try:
-        return filename, open(filename, mode).read()
-    except IOError as exc:
-        raise argparse.ArgumentTypeError(exc.strerror)
+    return filename, open(filename, mode).read()
```

Comment:
I actually want to go in the opposite direction. If do the work upfront to ensure absolute paths are used, we can go through and remove some of the 26 calls to `os.path.abspath` between the `letsencrypt`, `letsencrypt-apache`, `letsencrypt-nginx` modules.